<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Spheres</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>450</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-2</y>
             <width>240</width>
             <height>200</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>600</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ListMarker" name="markerList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>1024</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="newSphere">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>New sphere...</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/feather/highlight.svg</normaloff>:/icons/feather/highlight.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newAlign">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>New align region...</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/feather/align.svg</normaloff>:/icons/feather/align.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newMeasure">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>New measure...</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/feather/ruler.svg</normaloff>:/icons/feather/ruler.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newWhite">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>New white balance region...</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/feather/white.svg</normaloff>:/icons/feather/white.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Images</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="imageList1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>4</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>1024</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionLoad_LP"/>
    <addaction name="actionSave_LP"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionToggle_max_luma"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionView_RTI"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNewSphere"/>
    <addaction name="actionNewMeasure"/>
    <addaction name="actionNewWhite"/>
    <addaction name="actionNewAlign"/>
    <addaction name="separator"/>
    <addaction name="actionDetectHighlights"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_selected"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_all_right"/>
    <addaction name="actionRotate_all_right_2"/>
    <addaction name="actionDStretch"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>Calibration</string>
    </property>
    <addaction name="actionLens_parameters"/>
    <addaction name="actionWhite_balance"/>
    <addaction name="actionDome_geometry"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>&amp;Zoom...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>feather/zoom-in.svg</normaloff>feather/zoom-in.svg</iconset>
     </property>
     <addaction name="actionDeepzoom"/>
     <addaction name="actionTarzoom"/>
     <addaction name="actionItarzoom"/>
    </widget>
    <addaction name="actionExport_RTI"/>
    <addaction name="actionExport_Normals"/>
    <addaction name="actionConvert_rti"/>
    <addaction name="menuZoom"/>
    <addaction name="separator"/>
    <addaction name="actionShow_queue"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit_2"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCalibration"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="separator"/>
   <addaction name="actionExport_RTI"/>
   <addaction name="actionShow_queue"/>
   <addaction name="actionView_RTI"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/folder.svg</normaloff>:/icons/feather/folder.svg</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/chevron-right.svg</normaloff>:/icons/feather/chevron-right.svg</iconset>
   </property>
   <property name="text">
    <string>Next image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/chevron-left.svg</normaloff>:/icons/feather/chevron-left.svg</iconset>
   </property>
   <property name="text">
    <string>Previous image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/zoom-in.svg</normaloff>:/icons/feather/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/zoom-out.svg</normaloff>:/icons/feather/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionNewSphere">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/highlight.svg</normaloff>:/icons/feather/highlight.svg</iconset>
   </property>
   <property name="text">
    <string>Add a sphere</string>
   </property>
   <property name="toolTip">
    <string>New sphere...</string>
   </property>
  </action>
  <action name="actionDelete_selected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete selected item</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/info.svg</normaloff>:/icons/feather/info.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDetectHighlights">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/settings.svg</normaloff>:/icons/feather/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Find highlights</string>
   </property>
  </action>
  <action name="actionSave_LP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/save.svg</normaloff>:/icons/feather/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save LPs...</string>
   </property>
  </action>
  <action name="actionExport_RTI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/download.svg</normaloff>:/icons/feather/download.svg</iconset>
   </property>
   <property name="text">
    <string>Export RTI...</string>
   </property>
  </action>
  <action name="actionLoad_LP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/folder.svg</normaloff>:/icons/feather/folder.svg</iconset>
   </property>
   <property name="text">
    <string>Load LP...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/save.svg</normaloff>:/icons/feather/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="toolTip">
    <string>Save project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/file.svg</normaloff>:/icons/feather/file.svg</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>New project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewMeasure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/ruler.svg</normaloff>:/icons/feather/ruler.svg</iconset>
   </property>
   <property name="text">
    <string>Set scale</string>
   </property>
   <property name="toolTip">
    <string>Pick a known length and set dataset scale.</string>
   </property>
  </action>
  <action name="actionToggle_max_luma">
   <property name="text">
    <string>Toggle max luma</string>
   </property>
  </action>
  <action name="actionLens_parameters">
   <property name="text">
    <string>Lens parameters...</string>
   </property>
  </action>
  <action name="actionRotate_all_right">
   <property name="text">
    <string>Rotate right all images</string>
   </property>
  </action>
  <action name="actionRotate_all_right_2">
   <property name="text">
    <string>Rotate left all images</string>
   </property>
  </action>
  <action name="actionView_RTI">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/cast.svg</normaloff>:/icons/feather/cast.svg</iconset>
   </property>
   <property name="text">
    <string>View RTI </string>
   </property>
   <property name="toolTip">
    <string>View RTI in the browser</string>
   </property>
  </action>
  <action name="actionNormals">
   <property name="text">
    <string>Normals</string>
   </property>
  </action>
  <action name="actionExport_RTI_2">
   <property name="text">
    <string>Export RTI...</string>
   </property>
  </action>
  <action name="actionExport_Normals">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Normals...</string>
   </property>
  </action>
  <action name="actionShow_queue">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/list.svg</normaloff>:/icons/feather/list.svg</iconset>
   </property>
   <property name="text">
    <string>Show queue...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/settings.svg</normaloff>:/icons/feather/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionWhite_balance">
   <property name="text">
    <string>White balance...</string>
   </property>
  </action>
  <action name="actionDome_geometry">
   <property name="text">
    <string>Dome geometry...</string>
   </property>
  </action>
  <action name="actionConvert_rti">
   <property name="text">
    <string>Convert .rti...</string>
   </property>
  </action>
  <action name="actionNewWhite">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/white.svg</normaloff>:/icons/feather/white.svg</iconset>
   </property>
   <property name="text">
    <string>New white balance sample</string>
   </property>
  </action>
  <action name="actionNewAlign">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/align.svg</normaloff>:/icons/feather/align.svg</iconset>
   </property>
   <property name="text">
    <string>New align marker</string>
   </property>
  </action>
  <action name="actionDeepzoom">
   <property name="text">
    <string>Deepzoom</string>
   </property>
  </action>
  <action name="actionTarzoom">
   <property name="text">
    <string>Tarzoom</string>
   </property>
  </action>
  <action name="actionItarzoom">
   <property name="text">
    <string>Itarzoom</string>
   </property>
  </action>
  <action name="actionDStretch">
   <property name="text">
    <string>D-Stretch</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListMarker</class>
   <extends>QFrame</extends>
   <header>qmarkerlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
